src/js/health-records/components/ErrorableRadioButtons.jsx

```diff
@@ -0,0 +1,188 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import _ from 'lodash';
+import classNames from 'classnames';
+
+import ToolTip from '../../common/components/form-elements/ToolTip';
+import ExpandingGroup from '../../common/components/form-elements/ExpandingGroup';
+
+import { makeField } from '../../common/model/fields.js';
+
+/**
+ * A radio button group with a label.
+ *
+ * Validation has the following props.
+
+ * `additionalFieldsetClass` - String for any additional fieldset classes.
+ * `additionalLegendClass` - String for any additional legend classes.
+ * `label` - String for the group field label.
+ * `name` - String for the name attribute.
+ * `toolTipText` - String with help text for user.
+ * `tabIndex` - Number for keyboard tab order.
+ * `options` - Array of options to populate group.
+ * `required` - is this field required.
+ * `value` - string. Value of the select field.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+
+// NOTE: This component has been modified specifically for 508 compliance on the health records app
```

Comment:
What was the modification? Is it that it uses `children` to render anything nested within the component? That should be made clearer.

Reply:
Specifically if additional content like those date boxes need to be displayed, it cannot be nested in the label and should be outside of it instead. I'll make some changes here to get it working in the way it was intended